lightning_client.go

```diff
@@ -3619,6 +3625,20 @@ func (s *lightningClient) ChannelAcceptor(ctx context.Context,
 	return errChan, nil
 }
 
+// FundingStateStep is a funding related call that allows the execution
+// of some preparatory steps for a funding workflow or manual
+// progression of a funding workflow.
+func (s *lightningClient) FundingStateStep(ctx context.Context,
+	req *lnrpc.FundingTransitionMsg) (*lnrpc.FundingStateStepResp, error) {
+
+	resp, err := s.client.FundingStateStep(ctx, req)
```

Comment:
In order to make this call succeed you also need to wrap the context, first with a timeout context then also add permissions:

```
rpcCtx, cancel := context.WithTimeout(ctx, s.timeout)
defer cancel()
rpcCtx = s.adminMac.WithMacaroonAuth(rpcCtx)
```